src/main/java/org/tugraz/sysds/parser/BuiltinFunctionExpression.java

```diff
@@ -1715,6 +1722,14 @@ protected void checkMatrixParam(Expression e) {
 		}
 	}
 
+	protected void checkMatrixTensorParam(Expression e) {
+		if (e.getOutput().getDataType() != DataType.MATRIX && e.getOutput().getDataType() != DataType.TENSOR) {
+			raiseValidateError(
+					"Expected " + e.getText() + " to be a matrix or tensor argument for function " + this.getOpCode().toString().toLowerCase() + "().",
+					false);
```

Comment:
I copied this from `checkMatrixParam` and adapted it, should I also reformat other methods that also? (In general I tried to stay as close as possible to already implemented methods, should I adapt them if I think it would lead to more readable code?)